apps/designer/app/designer/designer.tsx

```diff
@@ -265,6 +265,8 @@ export type DesignerProps = {
   treeId: string;
   buildId: string;
   buildOrigin: string;
+  readToken: string;
+  sharedTokens: { token: string; relation: "viewers" | "editors" | "owner" }[];
```

Comment:
Let's give them additional naming to avoid confusion with design tokens.
readAccessToken
sharedAccessTokens